WcaOnRails/app/models/delegate.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class Delegate < ApplicationRecord
```

Comment:
It might be cleaner to call this `DelegateInfo`. I know in a lot of our tests, we do something like `delegate = FactoryGirl.create :user, :delegate`. The assumption is that a delegate is a particular kind of user. Creating a class called Delegate (as you've done here) would break that assumption, which I think will cause a lot of variable name changes.